functions/test/endpoints/PointLog-Tests/Handle.test.ts

```diff
@@ -50,39 +51,36 @@ describe('point_log/handle', async ()  => {
         await FirestoreDataFactory.setUser(db, PRIV_RES, 4)
         await FirestoreDataFactory.setUser(db, EA_ID, 5)
         await FirestoreDataFactory.setPointType(db, 1)
-        await FirestoreDataFactory.setPointType(db, 2, {residents_can_submit: false})
-        await FirestoreDataFactory.setPointType(db, 3, {is_enabled:false})
+        await FirestoreDataFactory.setPointType(db, 1, {description: PT_1_DETAILS, name: PT_1_DETAILS, value: 10})
+        await FirestoreDataFactory.setPointType(db, 2, {residents_can_submit: false, value: 10})
+        await FirestoreDataFactory.setPointType(db, 3, {is_enabled:false, value: 10})
         await FirestoreDataFactory.setHouse(db, HOUSE_NAME)
-        await FirestoreDataFactory.setHouseCode(db, HOUSE_CODE)
+
+        await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, false, {id: PARAM_CHECKS_LOG_ID})
+        await FirestoreDataFactory.setPointLog(db, HOUSE_NAME, RESIDENT_ID, true, {id: REJECT_LOG_ID, description: REJECT_LOG_ID, point_type_id: 1, approved: false})
     })
```

Comment:
Look at the description in the following comment for why you should do all data setup in the beforeAll